### ISSUE TITLE
Build docker image

### ISSUE THREAD

**Issue by ddcprg** [lyft/presto-gateway#152]
It would be useful to configuring a Docker build for this repo and publishing to `hub.docker.com` or some other public repo.

I've got a `Dockerfile` I'm using myself to build this, I'll be happy to raise a PR with this and the build settings if you think this can be useful.

**Comment by endoplasmicR** [comment]
Yes, that would be very useful, there are different base images people may elect to use, but a reference implementation would be valuable

**Comment by posulliv** [comment]
I wrote some helm charts I use for deploying the gateway and in the repo for the helm charts, I have a `Dockerfile` and scripts to build a docker image. You can see it here in case its useful - https://github.com/posulliv/lyft-gateway-charts
